README.md

```diff
@@ -1,2 +1,74 @@
-# oam-runtime
-A set of libraries for building OAM runtimes
+# oam-kubernetes-runtime Project
+
+The OAM Kubernetes runtime project is a collection of Golang help libraries and official api type
```

Comment:
My suggestion:

The OAM Kubernetes Runtime is the building block of data model in [Crossplane](https://github.com/crossplane/crossplane). It's basically a collection of Golang help libraries and api types to make implementing [Open Application Model](https://github.com/oam-dev/spec) easier on Kubernetes. 

OAM Kubernetes Runtime is designed to be platform builder facing and embedded into a larger system, for end-users facing experience of OAM, please always check [Crossplane official documentation](https://crossplane.github.io/docs).